src/web/components/Core/PagingToolHelper.ts

```diff
@@ -0,0 +1,39 @@
+const rowsPerPageValues = [10, 25, 50, 100, 250] as const;
+
+export type RowsPerPageValues = (typeof rowsPerPageValues)[number];
+
+export const getPageNumberOptions = (numberTotalRows: number, rowsPerPage: RowsPerPageValues) => {
+  return Array.from(
+    { length: Math.ceil(numberTotalRows / rowsPerPage) },
+    (_, index) => index + 1
+  ).map((number) => ({
+    name: number.toString(),
+    id: number,
+  }));
+};
+
+export const getRowsPerPageOptions = (numberTotalRows: number) => {
+  const rowsPerPageOptions: RowsPerPageValues[] = [];
+  let i = 0;
+  while (i < rowsPerPageValues.length) {
+    rowsPerPageOptions.push(rowsPerPageValues[i]);
+    if (numberTotalRows <= rowsPerPageValues[i]) break;
+    i += 1;
+  }
+  return rowsPerPageOptions.map((value) => ({
```

Comment:
could this possibly return a `RowsPerPageOptions` type that has a `string` and a `RowsPerPageValues`? I'm finding it hard to understand what `name` and `id` mean here, especially since the same pattern is used for the page number options, so a similar comment applies to that.

Reply:
This is related to the SelectOption type that the SelectDropdown and MultiselectDropdowns use- the SelectOption type has the format - SelectOption<TValue> is { id: TValue, name: string}

Reply:
ah - so can it use the `SelectOption` type then?

Reply:
Done